satpy/readers/seviri_l1b_hrit.py

```diff
@@ -357,10 +372,17 @@ def read_epilogue(self):
             data = np.fromfile(fp_, dtype=hrit_epilogue, count=1)
             self.epilogue.update(recarray2dict(data))
 
+    def reduce(self, max_size):
```

Comment:
This is nearly exactly the same as the previous function called `reduce`. Can they be factorised ?